internal/admin/dashboard/templates/index.html

```
@@ -231,157 +236,371 @@ <h2>Registered Models</h2>
     </div>
 
     <!-- Filter -->
-    <div class="table-toolbar" x-show="models.length > 0 || modelFilter">
-        <input type="text" placeholder="Filter by model, provider, or owner..." x-model="modelFilter" class="filter-input">
+    <div class="table-toolbar" x-show="displayModels.length > 0 || modelFilter || aliasesAvailable">
+        <input type="text" placeholder="Filter by provider/model, alias, or owner..." x-model="modelFilter" class="filter-input">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)